include/mbedtls/ssl_ciphersuites.h

```diff
@@ -31,14 +31,27 @@
 extern "C" {
 #endif
 
+#if defined(MBEDTLS_DEPRECATED_WARNING)
+
+#define MBEDTLS_DEPRECATED __attribute__((deprecated))
+MBEDTLS_DEPRECATED typedef int deprecated_ciphersuite_id_t;
+#define MBEDTLS_DEPRECATED_CIPHERSUITE_ID( ID )        \
+    ( (deprecated_ciphersuite_id_t) ( ID ) )
+
+#else
+#define MBEDTLS_DEPRECATED_CIPHERSUITE_ID( ID ) ID
+#endif /* MBEDTLS_DEPRECATED_WARNING */
+
 /*
  * Supported ciphersuites (Official IANA names)
  */
 #define MBEDTLS_TLS_RSA_WITH_NULL_MD5                    0x01   /**< Weak! */
 #define MBEDTLS_TLS_RSA_WITH_NULL_SHA                    0x02   /**< Weak! */
 
-#define MBEDTLS_TLS_RSA_WITH_RC4_128_MD5                 0x04
-#define MBEDTLS_TLS_RSA_WITH_RC4_128_SHA                 0x05
+#define MBEDTLS_TLS_RSA_WITH_RC4_128_MD5 \
```

Comment:
I'd prefer these definitions to be in a single line for grepping. Unlike regular lines of codes, series of simple constant definitions like this don't suffer much if some lines are wrapped in casual reading. But when grepping, it's a pain to get the half you were searching and not the half you were trying to look up.

Reply:
Done.